metricbeat/module/ceph/health/_meta/fields.yml

```diff
@@ -0,0 +1,69 @@
+- name: health
+  type: group
+  description: >
+    health
+  fields:
+    - name: cluster.overall_status
+      type: keyword
+      description: >
+        Overall status of the cluster
+    - name: cluster.timechecks.epoch
+      type: long
+      description: >
+        Map version
+    - name: cluster.timechecks.round
+      type: long
+      description: >
+        timecheck round
+    - name: cluster.timechecks.round_status
+      type: keyword
+      description: >
+        Status of the round
+    - name: mon.avail_percent
+      type: long
+      description: >
+        Available percent of the MON
+    - name: mon.health
+      type: keyword
+      description: >
+        Health of the MON
+    - name: mon.kb_avail
+      type: long
+      description: >
+        Available KB of the MON
+    - name: mon.kb_total
+      type: long
+      description: >
+        Total KB of the MON
+    - name: mon.kb_used
+      type: long
+      description: >
+        Used KB of the MON
+    - name: mon.last_updated
+      type: date
+      description: >
+        Time when was updated
+    - name: mon.name
+      type: keyword
+      description: >
+        Name of the MON
+    - name: mon.store_stats.bytes_log
+      type: long
+      description: >
+        Log bytes of MON
+    - name: mon.store_stats.bytes_misc
```

Comment:
mon.store_stats.misc.bytes

You can also add `format: bytes` to all bytes value, so Kibana will automatically know how to convert it to MB etc.

```
    - name: mon.store_stats.sst.bytes
      type: long
      description: >
        SST bytes of MON
      format: bytes
```